deltachat-ios/Controller/AccountSetup/WelcomeViewController.swift

```diff
@@ -216,13 +216,9 @@ class WelcomeViewController: UIViewController {
         } else {
             guard let permille = ui["progress"] as? Int else { return }
             var statusLineText = ""
-            if permille <= 100 {
-                statusLineText = String.localized("preparing_account")
-            } else if permille <= 950 {
-                let percent = ((permille-100)*100)/850
+            if permille < 1000 {
+                let percent: Int = permille/10
```

Comment:
here, permille 0 and 1000 are handled by special booleans 'done' and 'error' above; that could get some streamlining as well, but i wanted to keep this PR on point